_opensearch/snapshot-restore.md

```diff
@@ -156,8 +156,23 @@ Setting | Description
    s3.client.default.proxy.port: 8080 # port for your proxy server
    s3.client.default.read_timeout: 50s # the S3 connection timeout
    s3.client.default.use_throttle_retries: true # whether the client should wait a progressively longer amount of time (exponential backoff) between each successive retry
+   s3.client.default.region: us-east-2 # AWS region to use
    ```
 
+1. (Optional) If you don't want to use AWS access and secret keys, you could configure S3 plugin to use AWS IAM roles for service accounts:
```

Comment:
Insert "the" following "configure". If this is its first appearance, IAM should be defined as "AWS Identity and Access Management (IAM)". For subsequent appearances, use only "IAM" (do not use "AWS IAM").

Reply:
:+1: